examples/asr/conf/conformer/conformer_ctc_bpe.yaml

```diff
@@ -60,7 +58,6 @@ model:
     shuffle: false
     num_workers: 8
```

Comment:
I wonder what the logic is behind using 8 workers. Is this an optimal number? These 8 workers will be for each GPU worker, which seems excessive

Reply:
Depends on number of GPUs. Should we make it 6 or 4?
@titu1994

Reply:
Right it's 8 per GPU, 64 on most dgx. Dgx usually has 128 to 256 physical cores (depending on V100 vs A100) so 64 workers should be ok.